src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -91,6 +94,17 @@ public void initializeMotorControllers(){
     leftMaster.getEncoder().setPosition(0);
     rightMaster.getEncoder().setPosition(0);
     
+    leftMaster.setOpenLoopRampRate(0.5);
+    rightMaster.setOpenLoopRampRate(0.5);
+
+    leftMaster.setSmartCurrentLimit(20);
+    rightMaster.setSmartCurrentLimit(30);
+    right1.setSmartCurrentLimit(30);
+    right2.setSmartCurrentLimit(30);
+    left1.setSmartCurrentLimit(20);
+    left2.setSmartCurrentLimit(20);
```

Comment:
Sam, this is preventing brown-out very well right now, but it may need tweaking if we start to see issues down the line.